tests/simple/main.cpp

```diff
@@ -510,6 +510,67 @@ int main() {
                             .SetCompressionMethod(CompressionMethod::LZ4));
             RunTests(client);
         }
+
+        {
+            std::cout << "test 1" << std::endl;
```

Comment:
Oo, I deleted this line before review, but fogot to commit it. I think I would better to leave tests without names, because previous tests don't have names.

Reply:
Please use something like "test 1: <some name>"